internal/core/src/segcore/ChunkedSegmentSealedImpl.cpp

```diff
@@ -100,7 +102,8 @@ ChunkedSegmentSealedImpl::LoadVecIndex(const LoadIndexInfo& info) {
     AssertInfo(info.index_params.count("metric_type"),
                "Can't get metric_type in index_params");
     auto metric_type = info.index_params.at("metric_type");
-    auto row_count = info.index->Count();
+    auto accessor = SemiInlineGet(info.cache_index->PinCells({0}));
```

Comment:
seems here we pre-load the index only to verify row_count? can we avoid this so we can actually lazy load the index?

Reply:
The row_count can not be gotten before index loading. Let's just remove the check